pkg/sip/outbound.go

```diff
@@ -302,7 +304,11 @@ func (c *outboundCall) close(err error, status CallStatus, description string, r
 
 		// Call the handler asynchronously to avoid blocking
 		if c.c.handler != nil {
-			go c.c.handler.OnCallEnd(context.Background(), c.state.callInfo, description)
+			go c.c.handler.OnSessionEnd(context.Background(), &CallIdentifier{
+				ProjectID: c.projectID,
+				CallID:    c.state.callInfo.CallId,
+				SipCallID: c.state.callInfo.ParticipantAttributes[AttrSIPCallIDFull],
```

Comment:
It should be available as a field in the call itself. If not, feel free to add it. Please avoid getting things from attribute map.